CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [0.0.3] - 2022-11-02
+
+### Added
+
+- VSphere cluster provider
+- Builder macro for templated values in an agents config
+- EKS cluster provider accepts `eksctl` configuration files
+- ECS cluster provider support for existing IAM instance profiles
+- TestManager support for custom status columns
+- Support for bash agents using `test-agent-cli`
+
+### Changed
+
+- EC2 instance provider takes a list of instance types and subnets instead of one
+
+### Removed
+
+- Yamlgen was removed
```

Comment:
Can you please add links to the PRs that are associated with the change? Like we do in bottlerocket's change log.